Classes/Package.php

```diff
@@ -12,6 +12,8 @@
  */
 class Package extends BasePackage
 {
+    const NODE_TYPE_IDENTIFIER_MIXIN = 'Neos.Form.Builder:IdentifierMixin';
```

Comment:
good call to make this a constant, but IMO it should be private
```suggestion
    private const NODE_TYPE_IDENTIFIER_MIXIN = 'Neos.Form.Builder:IdentifierMixin';
```